README.md

```diff
@@ -10,71 +10,48 @@ The SRecord package is a collection of powerful tools for manipulating EPROM loa
-* **VHDL**: It is possible to write VHDL file. This is only supported for output.
-* **Verilog VMEM**:
-  It is possible to write a Verilog VMEM file suitable for loading with `$readmemh()`.
-  This format is supported for reading and writing.
-* **Wilson**:
-  The Wilson format is understood for both reading and writing.
-  This mystery format was added for a mysterious type of EPROM writer.
+* **Ascii-Hex**: input/output; also known as **ascii-space-hex**
+* **ASM**: output only; for embedding data into *Assembler* code
+* **Atmel Generic**: input/output; used by *Atmel AVR assembler*
+* **BASIC**: output only; for embedding data into *BASIC* code
+* **BinaryBinary**: input/output
+* **B-Record**: input/output; also known as **Freescale Dragonball bootstrap b-record**
+* **C**: output only; for embedding data into *C* code
+* **COE**: output only; Extension `.coe`; also known as **Xilinx Coefficient File** format
+* **Cosmac**: input/output; also known as **RCA Cosmac Elf** format
+* **DEC Binary (XXDP)**: input/output
+* **Elektor Monitor (EMON52)**: input/output
+* **Fairchild Fairbug**: input/output
+* **Formatted Binary**: input/output
+* **Four Packed Code (FPC)**: input/output
+* **Hexdump**: output only; a simple hexdump
+* **HP64000 Absolute**: input only
+* **IDT/sim**: input/output
+* **Intel**: input/output; also known as **Intel MCS-86 Object** format
+* **Intel Absolute Object Module Format (AOMF)**: input/output
+* **Intel 16 (INHX16)**: input/output; also known as **Intel hexadecimal 16**
+* **LSI Logic Fast Load**: input/output
+* **Logisim**: input/output
+* **Memory Initialization Format**: output only; Extension `.mem`; used by *Lattice Semiconductor*
+* **MIF**: input/output; used by *Altera* **Memory Initialization File** format
+* **MOS Technology**: input/output
+* **MIPS-Flash**: input/output
+* **Motorola S-Record**: input/output; also known as the **Exorciser**, **Exormacs** or **Exormax**
+* **MsBin**: input/output; also known as **Windows CE Binary Image Data** format
+* **Needham**: input/output; also known as **Needham Electronics ASCII file** format
+* **OS65V**: input/output; also known as **Ohio Scientific hexadecimal** format
+* **PPB**: input/output, also known as **Stag Prom Programmer binary**
+* **PPX**: input/output, also known as **Stag Prom Programmer hexadecimal**
+* **Signetics**: input/output
+* **SPASM**: input/output; used by a variety of *PIC* programmers
+* **Spectrum**: input/output
+* **Tektronix**: input/output
+* **Tektronix Extended**: input/output
+* **Texas Instruments Tagged**: input/output (both 8 and 16 bit); also known as **TI-tagged** or **TI-SDSMAC**
+* **Texas Instruments ti-txt**: input/output; used by bootstrap loader of *TI MSP430*
+* **TRS-80**: input/output; used by *The Radio Shack*
+* **VHDL**: output only
+* **Verilog VMEM**: input/output; suitable for loading with `$readmemh()`
+* **Wilson**: input/output; mysterious type of EPROM writer
```

Comment:
In the previous commit I just fixed some issues on the README.
In this commit I rewrote the text completely. I think this compact version better suits a README.
Please say which version to keep.

Reply:
I do like yours more. The readme was a last minute thing I added quickly as github "just looked wrong" without it. I briefly tried to generate it from troff but wasn't worth the effort so I copied from the webpage. Like I said, this is much more appealing.